### DISCUSSION TITLE
How to specify username that contains @ char ?

### DISCUSSION THREAD

**Discussion by bigluds** [kingToolbox/WindTerm#512] [Q&A] [answered]
I have a username that contains special chars '@' and '=', how to specify username that contains '@' and '=' chars in the Host field of the Session Settings window ?
Please note that this username is provided by an external enterprise and we are not able to change the username.

![image](https://user-images.githubusercontent.com/12299642/153963661-42c866d5-81c8-4c68-bff9-ace40a275aa1.png)

**Comment by kingToolbox** [comment]
Unfortunately, usernames containing the `@` character are not currently supported. I will implement this in the next `WindTerm_2.4.0` version. `WindTerm_2.4.0_Prerelease_1` is likely to be released this week, Please stay tuned, thank you.

**Reply by bigluds** [reply]
Many thanks for this tool and your reactivity. This correction will be greatly appreciated.

**Reply by kingToolbox** [reply]
I am happy to inform you that [WindTerm_2.3.1](https://github.com/kingToolbox/WindTerm/releases/tag/2.3.0) has been released and this issue has been fixed. Please download and check it. If there are still bugs, please let me know, thank you.

**Comment by bigluds** [comment, marked as answer]
Many thanks, after my fiesta test it seems working.

**Reply by kingToolbox** [reply]
Good news, please let me know if you still encounter any problems, thank you.
